env/edit.go

```diff
@@ -106,3 +107,43 @@ func parseVariable(param string) (string, string) {
 	editSplit := strings.SplitN(param, "=", 2)
 	return editSplit[0], editSplit[1]
 }
+
+func readFromCmdLine(variables *scalingo.Variables, params []string) error {
+	for _, param := range params {
+		if err := isEnvEditValid(param); err != nil {
+			return errgo.Newf("'%s' is invalid: %s", param, err)
+		}
+
+		name, value := parseVariable(param)
+		*variables = append(*variables, &scalingo.Variable{
+			Name:  name,
+			Value: value,
+		})
+	}
+	return nil
+}
+
+func readFromFile(variables *scalingo.Variables, filePath string) error {
+	if len(filePath) > 0 {
```

Comment:
best practice: we prefer exiting the function as early as possible.
```suggestion
	if len(filePath) == 0 {
		return nil
	}
```